tests/integration-tests-isolated/src/test/java/org/apache/activemq/artemis/tests/integration/isolated/web/WebServerComponentTest.java

```diff
@@ -121,7 +121,7 @@ public void testRequestLog() throws Exception {
       ch.writeAndFlush(request);
       assertTrue(latch.await(5, TimeUnit.SECONDS));
       assertEquals("12345", clientHandler.body.toString());
-      assertEquals(clientHandler.body.toString(), "12345");
+      assertEquals("12345", clientHandler.body.toString());
```

Comment:
Assert on line above appears exactly the same, so this could perhaps be removed (or changed, if it wasnt just a c&p error, but it seems likely thats what it was)